backend/src/router_test.go

```diff
@@ -515,6 +515,30 @@ func (suite *RouterSuite) TestPatchProfile() {
 	}))
 }
 
+func (suite *RouterSuite) TestPutTutorsUsername() {
+	suite.Run("Success", manualSetupTest(func() {
+		hash, _ := bcrypt.GenerateFromPassword([]byte("Password"), bcrypt.DefaultCost)
+		DB.Create(&User{Username: "Username", Password: string(hash)})
+
+		w := POST("/signin", gin.H{
+			"username": "Username",
+			"password": "Password",
+		})
+		suite.Equal(200, w.Code)
+		cookies := w.Result().Cookies()
```

Comment:
We don't need to check cookies for this test, only the authentication routes need to worry about that. Checking the status is good enough to at least make sure the request didn't fail.

Reply:
Removed it